### PR TITLE
Added network options

### PR DESCRIPTION
**Pull request by MartinSchmidt**
Changed Vault to support the NetworkOptions
Also updated the referenced images of registry and verifier

### CODE REVIEW

File: src/ProjectOrigin.Vault/Activities/SendRegistryTransactionActivity.cs [hunk 1]
```diff
@@ -16,10 +16,10 @@ public record SendRegistryTransactionArguments
 
 public class SendRegistryTransactionActivity : IExecuteActivity<SendRegistryTransactionArguments>
 {
-    private readonly IOptions<RegistryOptions> _registryOptions;
+    private readonly IOptions<NetworkOptions> _registryOptions;
     private readonly ILogger<SendRegistryTransactionActivity> _logger;
 
-    public SendRegistryTransactionActivity(IOptions<RegistryOptions> registryOptions, ILogger<SendRegistryTransactionActivity> logger)
+    public SendRegistryTransactionActivity(IOptions<NetworkOptions> registryOptions, ILogger<SendRegistryTransactionActivity> logger)
```
Comment (tnickelsen) [inline src/ProjectOrigin.Vault/Activities/SendRegistryTransactionActivity.cs:22]: call this networkOptions

File: src/ProjectOrigin.Vault/Activities/WaitCommittedRegistryTransactionActivity.cs [hunk 2]
```diff
@@ -25,11 +25,11 @@ public record WaitCommittedTransactionArguments
 
 public class WaitCommittedRegistryTransactionActivity : IExecuteActivity<WaitCommittedTransactionArguments>
 {
-    private readonly IOptions<RegistryOptions> _registryOptions;
+    private readonly IOptions<NetworkOptions> _registryOptions;
     private readonly ILogger<WaitCommittedRegistryTransactionActivity> _logger;
     private readonly IUnitOfWork _unitOfWork;
 
-    public WaitCommittedRegistryTransactionActivity(IOptions<RegistryOptions> registryOptions, ILogger<WaitCommittedRegistryTransactionActivity> logger, IUnitOfWork unitOfWork)
+    public WaitCommittedRegistryTransactionActivity(IOptions<NetworkOptions> registryOptions, ILogger<WaitCommittedRegistryTransactionActivity> logger, IUnitOfWork unitOfWork)
```
Comment (tnickelsen) [inline src/ProjectOrigin.Vault/Activities/WaitCommittedRegistryTransactionActivity.cs:32]: callthis networkoptions

File: chart/run_kind_test.sh [hunk 3]
```diff
@@ -73,6 +78,16 @@ rabbitmq:
     secretRef:
       name: rabbitmq
       key: rabbitmq-password
+
+networkConfig:
```
Comment (tnickelsen) [inline chart/run_kind_test.sh:82]: What about stamps?

File: test/ProjectOrigin.Vault.Tests/TestExtensions/IWaitForContainerOSExtensions.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Net;
+using DotNet.Testcontainers.Configurations;
+
+namespace ProjectOrigin.Vault.Tests.TestExtensions;
+
+public static class IWaitForContainerOSExtensions
+{
+    public static IWaitForContainerOS UntilGrpcResponds(this IWaitForContainerOS waitForContainer, ushort grpcPort, Action<IWaitStrategy>? waitStrategyModifier = null)
```
Comment (tnickelsen) [inline test/ProjectOrigin.Vault.Tests/TestExtensions/IWaitForContainerOSExtensions.cs:9]: skal dette i common librariet?

Comment (MartinSchmidt) [reply test/ProjectOrigin.Vault.Tests/TestExtensions/IWaitForContainerOSExtensions.cs]: done :)

**Review by tnickelsen — APPROVED** [review]

**Pull request merged**